datacapture/src/test/java/com/google/android/fhir/datacapture/MoreQuestionnaireItemComponentsTest.kt

```diff
@@ -487,6 +488,51 @@ class MoreQuestionnaireItemComponentsTest {
     assertThat(questionnaire.mimeTypes).isEmpty()
   }
 
+  @Test
+  fun maxSize_shouldReturnBytes() {
+    val questionnaire =
```

Comment:
```suggestion
    val questionnaireItem =
```

and below. it's just a bit confusing to call this a questionnaire when it's not.

also i think these can all be inlined.

Reply:
sure, renamed.
I inlined the null type tests.